nextflow.config

```diff
@@ -59,6 +59,11 @@ params {
     validationShowHiddenParams       = false
     validate_params                  = true
 
+    // ExomeDepth Parameters
+    roi_auto = "/home/torossee/projects/nf-cmgg-exomecnv_references/Homo_sapiens.GRCh38.105.chr_protein_coding_basic_sorted_merged_autosomal.bed"
+    roi_chrx = "/home/torossee/projects/nf-cmgg-exomecnv_references/Homo_sapiens.GRCh38.105.chrX_protein_coding_basic_sorted_merged.bed"
```

Comment:
These defaults should be `null`. You can add path to these files in the `conf/test.config` so you can succesfully run the tests. (Please also add these files to github.com/nf-cmgg/test-datasets` so a URL can be used instead of a local path, now this pipeline will only work on your machine)

Reply:
ok, I will remove this in the next PR

Reply:
and I will add the files to the test-dataset repo!

Reply:
resolved